.github/workflows/ci.yml

```
@@ -0,0 +1,22 @@
+on: ["push"]
+
+jobs:
+    arduino:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: arduino/actions/libraries/compile-examples@master
```

```suggestion
            - uses: arduino/compile-sketches@main
```
The action has been moved to a dedicated repository:
https://github.com/arduino/compile-sketches

The original experimental version of the action in the original experimental https://github.com/arduino/actions repository you were using is no longer maintained. So it's recommended to use the actively maintained `arduino/compile-sketches` instead.

---

I hope the syntax didn't change 😬

---

@per1234 I think there is no link to https://github.com/arduino/compile-sketches in the README of https://github.com/arduino/actions/. Maybe add that?

---

A format deprecation process is planned for the old action version. This is only waiting on the production release of the `arduino/compile-sketches` action.

Even though this transition is not complete, as the developer of this tool I can tell you with confidence that you're better off using `arduino/compile-sketches` than `arduino/actions/libraries/compile-examples` at this point.

---

Perfect! Thank you a lot. This action is very handy as a quick minimal ci xD

---

Arduino has now made the 1.0.0 release of the `arduino/compile-sketches` action. If you like, you can use the release version of the action by changing this:
```yaml
- uses: arduino/compile-sketches@main
```
to this
```yaml
- uses: arduino/compile-sketches@v1
```
The `@main` will continue to work fine though, so it's up to you. The difference is the `@v1` ref will cause the release version of the action to be used, whereas the `@main` ref will cause the beta version of the action to be used. They are currently identical, but as development work is done on the action, the `@main` will immediately integrate it, whereas the `@v1` will only do so once a new release is created. So `@main` provides the latest and greatest, while `@v1` provides a more stable version of the action.

This allowed me to formally deprecate the experimental `arduino/actions/libraries/compile-examples` version of the action. This includes adding a link to the new action in the root readme of that repo, the action's readme, and also making the action display a warning with the link to the dedicated repo.
https://github.com/arduino/actions/pull/92
I think this resolved all the excellent suggestions you have made here. Please feel free to let me know if you have any others (I already have added https://github.com/arduino/actions/issues/91#issuecomment-811819511 on the "to do" list).